### ISSUE TITLE
Heading syntax highlighting mismatched with preview/rendering

### ISSUE THREAD

**Issue by pepijnve** [asciidoctor/asciidoctor-intellij-plugin#542]
In the following document

```
= Title

== H1

=== H2

* list item 1.
** nested list item 2

=== Another H2

Content
```

`Another H2` is not highlighted as a heading in the editor view and it does not show up in the structure view. The preview window however does render this as a heading.

<img width="482" alt="image" src="https://user-images.githubusercontent.com/225326/91741462-0b55c600-ebb5-11ea-94f8-68314c6edecd.png">
AsciiDoc Plugin Version: 0.31.22

IDE Name and Version: IntelliJ IDEA 2020.2.1 (Ultimate Edition) Build #IU-202.6948.69, built on August 24, 2020

**Comment by pepijnve** [comment]
I was just rereading the Asciidoctor user manual on lists. I think this might be caused by the 'attach to parent list item' behaviour of the list continuation. There is no list continuation (`+`) present here, so I don't think that logic should apply and the preview seems to agree.

**Comment by ahus1** [comment]
Thanks for reporting this. I fixed this in pre-release 0.31.23.

The pre-release of the plugin is available from [GitHub releases](https://github.com/asciidoctor/asciidoctor-intellij-plugin/releases) and the [IntelliJ AsciiDoc EAP repository](https://intellij-asciidoc-plugin.ahus1.de/docs/contributors-guide/user/add-eap-repository-to-ide.html).
